build.gradle

```diff
@@ -22,8 +22,15 @@ buildscript {
       intellij: "IC-$ideaVersion",
       jps: "JPS-$ideaVersion",
       okio: 'com.squareup.okio:okio:1.11.0',
-      intellijCore: fileTree(dir: 'lib/intellij-core/lib', include: ['*.jar']),
-      sqlitePsi: fileTree(dir: '/Users/astrong/Development/sqlite/sqlite-psi/core/build/libs', include: ['*.jar']),
+      intellijCore: fileTree(dir: 'lib/intellij-core/lib', include: [
+          'idea.jar', 'openapi.jar', 'util.jar', 'trove4j.jar', 'extensions.jar',
+          'picocontainer.jar', 'asm-all.jar', 'guava-19.0.jar', 'automaton.jar'
+      ]),
+      intellijGradleCore: fileTree(dir: 'lib/intellij-core/lib', include: [
+          'idea.jar', 'openapi.jar', 'util.jar', 'trove4j.jar', 'extensions.jar',
+          'picocontainer.jar', 'asm-all.jar', 'automaton.jar'
```

Comment:
the guava jar downloaded for intellij conflicts with AGP's guava, so I just dont depend on the local jar